### PR TITLE
add columns to table

### PR DESCRIPTION
**Pull request by nicflores**
Tables now have column descriptions.
This is needed to store column information sent to us by the frontend.

### CODE REVIEW

**Comment by nicflores** [comment]
Release snapshot `29.0.15-9d5048b`

File: api/src/main/scala/quasar/api/table/ColumnScalar.scala [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import slamdata.Predef.{Product, Serializable}
+
+sealed abstract class ColumnScalar extends Product with Serializable
+
+object ColumnScalar {
+  final case object Boolean extends ColumnScalar
+  final case object Null extends ColumnScalar
+  final case object Number extends ColumnScalar
+  final case object String extends ColumnScalar
+  final case object Offsetdatetime extends ColumnScalar
```
[the remark this answers is not in the document]
Comment (nicflores) [reply api/src/main/scala/quasar/api/table/ColumnScalar.scala]: Sure. I'll change it and then push the changes upstream to `mimir` and `sdbe`.

[the remark this answers is not in the document]
Comment (alissapajer) [reply api/src/main/scala/quasar/api/table/ColumnScalar.scala]: Great. I'll merge this as soon as the changes build.

**Comment by alissapajer** [comment]
Also, since this is breaking for mimir and sdbe, I'm going to merge this only when the mimir and sdbe PRs are both approved. That way we won't be blocking any upstream version upgrades.

**Pull request merged**